src/GafferSceneUI/CropWindowTool.cpp

```diff
@@ -459,6 +464,49 @@ void CropWindowTool::plugDirtied( const Gaffer::Plug *plug )
 	}
 }
 
+void CropWindowTool::nodeMetadataChanged( IECore::InternedString key, const Gaffer::Node* node )
+{
+	if( !m_cropWindowPlug || m_overlayDirty )
+	{
+		return;
+	}
+
+	if( node != m_cropWindowPlug->node() )
```

Comment:
Any node in the parent hierarchy for `m_cropWindowPlug` could cause us to become read only, because read-onliness is inherited. So we can't early-out like this - we'd need to use `!node->isAncestorOf()` instead. There is an argument for just assuming nothing about how read-onliness is specified/inherited though, and doing as the TransformTool does - just risk doing a little more work once in a while.

Reply:
Ahh great, thanks. Fixed in https://github.com/GafferHQ/gaffer/pull/3239/commits/558040aa1361bf1df48a471a037f3655bff30c97